application/controllers/teams_controller.py

```diff
@@ -28,7 +28,7 @@ class TeamsController(ApplicationController):
 
     def show(self, object_id):
         try:
-            team = Team.find_by(force=True, id_=object_id)
+            team = Team.find_by(force=True, id=object_id)
```

Comment:
The new neo model uses id instead of id_. As Adrian said, overriding the id member is not a problem.